src/main/c/javacan_bcmmessage.c

```diff
@@ -0,0 +1,71 @@
+/**
+ * The MIT License
+ * Copyright © 2018 Phillip Schichtel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+#include <tel_schich_javacan_BcmMessage.h>
+#include <stddef.h>
+#include <jni.h>
+#include <linux/can/bcm.h>
+
+JNIEXPORT jint JNICALL Java_tel_schich_javacan_BcmMessage_getLongSize(JNIEnv *env, jclass class) {
+    long dummy=0;
+    return sizeof(dummy);
```

Comment:
frankly... this contribution is my first C code since almost two decades! :smile:
I think i had an argument with the compiler on my first try... but you are right.
-> changed it.

Reply:
happens :)